libraries/AP_NavEKF2/AP_NavEKF2_Control.cpp

```diff
@@ -250,6 +250,9 @@ void NavEKF2_core::setAidingMode()
             tasTimeout = true;
             gpsNotAvailable = true;
         } else if (posAidLossCritical) {
+            if ((frontend->_flowUse & FLOW_USE_NAV) && optFlowDataPresent() && (imuSampleTime_ms - rngValidMeaTime_ms < 500)) {
```

Comment:
should be _flowUse == FLOW_USE_NAV I think?

Reply:
thank you @tridge 
I use & FLOW_USE_NAV because _flowUse is a bitmask  according to
https://github.com/ArduPilot/ardupilot/blob/dba272bafba851d7c8096a78306afd52620f0066/libraries/AP_NavEKF2/AP_NavEKF2.cpp#L561-L562

Reply:
OK, interesting.  The way it's used in other parts of the code is using the "==" but this is perhaps because for now we only have two options so either way works.  Let's keep this in mind but stay consistent (i.e. let's keep using "==" for now).

Reply:
I got it. thank you @rmackay9